pandas/tests/extension/base/reshaping.py

```diff
@@ -124,11 +124,11 @@ def test_merge(self, data, na_value):
                             'key': [0, 1, 2]})
         df2 = pd.DataFrame({'int2': [1, 2, 3, 4], 'key': [0, 0, 1, 3]})
 
-        res = pd.merge(df1, df2)
-        exp = pd.DataFrame(
-            {'int1': [1, 1, 2], 'int2': [1, 2, 3], 'key': [0, 0, 1],
-             'ext': data._from_sequence([data[0], data[0], data[1]])})
-        self.assert_frame_equal(res, exp[['ext', 'int1', 'key', 'int2']])
+        # res = pd.merge(df1, df2)
+        # exp = pd.DataFrame(
+        #     {'int1': [1, 1, 2], 'int2': [1, 2, 3], 'key': [0, 0, 1],
+        #      'ext': data._from_sequence([data[0], data[0], data[1]])})
+        # self.assert_frame_equal(res, exp[['ext', 'int1', 'key', 'int2']])
```

Comment:
this failed?

Reply:
No, was just debugging & forgot to uncomment.